helm/event-exporter-app/templates/pss-exceptions.yaml

```diff
@@ -1,4 +1,9 @@
-apiVersion: kyverno.io/v2alpha1
+{{- if .Capabilities.APIVersions.Has "kyverno.io/v2beta1/PolicyException" -}}
+{{- if .Capabilities.APIVersions.Has "kyverno.io/v2/PolicyException" -}}
```

Comment:
```suggestion
{{- if .Capabilities.APIVersions.Has "kyverno.io/v2beta1/PolicyException" -}}
{{- if .Capabilities.APIVersions.Has "kyverno.io/v2/PolicyException" -}}
```

We are probably not installing this before Kyverno, but it would just ignore the whole CR and never try again. So would you rather like to ignore it or fail hard?

Reply:
TBH it's not critical, I only want to ensure that we could apply v2 instead of v2beta1